backend/Controllers/RecipeController.js

```diff
@@ -488,4 +507,7 @@ const RecipeController = {
   deleteComment,
 };
 
-export default RecipeController;
+export default {
+  RecipeController,
+  getRandomRecipe,
+};
```

Comment:
@sanchitc05 add the function in the above exported object only

Reply:
Can you please elaborate

Reply:
There is a object RecipeControllet right above it, add the new function in that